src/TTextEdit.cpp

```diff
@@ -128,6 +128,9 @@ TTextEdit::TTextEdit(TConsole* pC, QWidget* pW, TBuffer* pB, Host* pH, bool isLo
     setEnabled(true);       //test fix for MAC
 
     connect(mpHost, &Host::signal_changeIsAmbigousWidthGlyphsToBeWide, this, &TTextEdit::slot_changeIsAmbigousWidthGlyphsToBeWide, Qt::UniqueConnection);
+    
+    if (mpHost->mAutoWrap)
```

Comment:
Minor code-style note - we use `{`...`}` around even a single line of the conditional code for an `if (....)`... - I'd post the code as a suggestion but my Web-browser is playing up and cannot process a multi-line selection currently... :roll_eyes:
```diff
-    if (mpHost->mAutoWrap)
+    if (mpHost->mAutoWrap) {
         mpBuffer->setWrapAt(width() / mFontWidth);
+    }
```